starter-code/index.js

```diff
@@ -1,10 +1,38 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+function SortedList() {
+  this.items = [];
+  this.length = 0;
+}
+
+SortedList.prototype.add = function(item) {
+  this.items.push(item);
+  this.items.sort(function(a, b) {
```

Comment:
Now with the new ES6 syntax, this could be written nicely as:
`
this.items.sort( (a, b) => a - b )
`